pkg/config/expand.go

```diff
@@ -199,11 +199,11 @@ func getDefaultGoogleProviders(bp Blueprint) map[string]TerraformProvider {
 	return map[string]TerraformProvider{
 		"google": {
 			Source:        "hashicorp/google",
-			Version:       ">= 4.84.0, < 5.45.0",
+			Version:       ">= 4.84.0, < 6.7.0",
```

Comment:
It's not obvious to me whether our changes may be backwards compatible to 4.84.0. The safest claim is that the final few releases of the 5.x branch are compatible because they were given the 6.x behavior while supporting the old behavior with deprecation warnings. They are still doing some backports on the 5.x release. I believe the appropriate target releases are now:

- https://github.com/hashicorp/terraform-provider-google/releases/tag/v5.44.2
- https://github.com/hashicorp/terraform-provider-google/releases/tag/v6.7.0

i.e. 
```suggestion
			Version:       ">= 5.44.2, < 6.8.0",
```